src/neptune_fetcher/read_only_project.py

```diff
@@ -161,6 +161,20 @@ def fetch_read_only_runs(
         for custom_id in custom_ids or []:
             yield ReadOnlyRun(read_only_project=self, custom_id=custom_id)
 
+    def fetch_read_only_experiments(
+        self,
+        names: Optional[List[str]] = None,
+    ) -> Iterator[ReadOnlyRun]:
+        """Lists experiments of the project in the form of read-only runs.
+
+        Returns a generator of `ReadOnlyRun` instances.
+
+        Args:
+            names: List of experiment names to fetch.
+        """
```

Comment:
```suggestion
        Example:
            ```
            project = ReadOnlyProject()
            for run in project.fetch_read_only_experiments(names=["yolo-v2", "yolo-v3"]):
                ...
            ```
        """
```